src/clients/permissions.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ */
+
+import * as config from "../configuration";
+import fetch from "node-fetch";
+
+export const getPublicAppIDs = async () => {
+    const reqURL = new URL(config.permissionsURL);
+    reqURL.pathname = `/permissions/subjects/group/${config.publicGroup}/app`;
+    const resp = await fetch(reqURL);
+    if (!resp.ok) {
+        const msg = await resp.text();
+        throw new Error(msg);
+    }
+    const resp_1 = resp;
```

Comment:
Is there a reason to create `resp_1` and not use `resp` directly?

Reply:
That's what I get for not reading an automated refactor before committing the change. 😊